x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/popover_tooltip.tsx

```diff
@@ -5,39 +5,61 @@
  * 2.0.
  */
 
-import React, { useState } from 'react';
-import { EuiPopover, EuiButtonIcon } from '@elastic/eui';
+import React, { useState, useCallback } from 'react';
+import type { EuiIconProps, IconColor } from '@elastic/eui';
+import { EuiIcon, EuiPopover, keys } from '@elastic/eui';
 import * as i18n from '../../../../detections/pages/detection_engine/rules/translations';
 
 interface PopoverTooltipProps {
   columnName: string;
   children: React.ReactNode;
+  colorAnchor?: IconColor;
```

Comment:
nit: if this is describing the color of an anchor tag (as opposed to a "colored anchor"):
```suggestion
  anchorColor?: IconColor;
```